src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -0,0 +1,1876 @@
+//------------------------------------------------------------------------------
+/*
```

Comment:
So far it looks like the only essential change is that a `Consensus` object is passed in to the `LedgerConsensusImp` constructor.  That allows `LedgerConsensusImp` to avoid some calls to `getApp().getOPs()`.